### ISSUE TITLE
requestExternalStorageAuthorization receive object in success callback parameter instead of string

### ISSUE THREAD

**Issue by kustolovic** [dpa99c/cordova-diagnostic-plugin#324]
**I'm submitting a bug report**:
- [x] bug report
- [ ] feature request
- [ ] documentation issue

# Bug report

**Current behavior:**

Since I update from 3.7.2 to 4.0.9, the success callback from requestExternalStorageAuthorization receive an object in the status parameter:
{"READ_EXTERNAL_STORAGE": "GRANTED"}

**Expected behavior:**

The status parameter should be a string (like 'GRANTED') as it was previously and described in the README.

maybe it is an uncompatible design change, it should be mentioned in the README and is a documentation issue.

**Steps to reproduce:**

```
cordova.plugins.diagnostic.requestExternalStorageAuthorization(function(status){
  console.log(JSON.stringify(status));
  if (status == cordova.plugins.diagnostic.permissionStatus.GRANTED) {
     console.log('GRANTED');  
  } else {
     console.log('NOT GRANTED');  
  }
}, function(error) { });
```

**Environment information**

- Cordova CLI version 
	- 7.1.0
- Cordova platform version
	- android 6.3.0
- Plugins & versions installed in project (including this plugin)
    - cordova-android-support-gradle-release 1.4.4 "cordova-android-support-gradle-release"
    - cordova-plugin-camera 4.0.3 "Camera"
    - cordova-plugin-device 2.0.2 "Device"
    - cordova-plugin-dialogs 2.0.1 "Notification"
    - cordova-plugin-file 6.0.1 "File"
    - cordova-plugin-file-opener2 2.0.19 "File Opener2"
    - cordova-plugin-file-transfer 1.7.1 "File Transfer"
    - cordova-plugin-globalization 1.11.0 "Globalization"
    - cordova-plugin-inappbrowser 3.0.0 "InAppBrowser"
    - cordova-plugin-whitelist 1.3.3 "Whitelist"
    - cordova.plugins.diagnostic 4.0.9 "Diagnostic"
    - phonegap-plugin-barcodescanner 8.0.0 "BarcodeScanner"
- Dev machine OS and version
    - macOs 10.12.6
        
_Runtime issue_
- Device details
    - Samsung Galaxy S4
    - Samsung Galaxy S6 Edge
- OS details
    - Android 5.0.1	
    - Android 7
	
_Android build issue:_	
- Node JS version
    - 5.10.1
- Gradle version
	- 4.10
- Target Android SDK version
	- 15
- Android SDK details

**Console output**

```
09-04 09:08:35.617: I/chromium(16494): [INFO:CONSOLE(4662)] "{"READ_EXTERNAL_STORAGE":"GRANTED"}", source: file:///android_asset/www/build/vendors.min.js (4662)
```

**Other information:**

I've tried to understand the cause in the code but I'm too limited in Java to find the root cause of the issue :)

**Comment by dpa99c** [comment]
I can reproduce this and confirm this regression but is present in v4.x of the plugin. 
It affects requesting all single permissions on Android.
I'll commit a fix and push it out in a patch release.

**Comment by dpa99c** [comment]
Fix is published in v4.0.10
